server/bin/unittests

```diff
@@ -61,23 +61,23 @@ function _run_re_unpack {
 }
 
 function _run_activate {
-    # Testing the server activate script, which is used for every other
-    # unit test as well is just a matter of emitting the resulting
-    # crontab along side the rest of the environment state provided by
-    # the unit test framework.
-    echo "+++ Verifying server activation" >> $_testout
-    sed -i 's;'${_gitdir}'/server/;/home/user/repo/pbench/server/;' $_testactout
-    cat $_testactout >> $_testout
-    echo "++++ ${_testopt}/lib/crontab/crontab" >> $_testout
-    cat ${_testopt}/lib/crontab/crontab >> $_testout
-    rc=$?
-    echo "---- ${_testopt}/lib/crontab/crontab" >> $_testout
-    if [ -s $_testactlog ]; then
-        echo "++++ $(basename $_testactlog) file contents" >> $_testout
-        cat $_testactlog >> $_testout 2>&1
-        echo "---- $(basename $_testactlog) file contents" >> $_testout
+    # Testing the server activate script, which is used for every other unit
+    # test as well is just a matter of emitting the resulting crontab along
+    # side the rest of the environment state provided by the unit test
+    # framework.
```

Comment:
I know you only changed the wrapping, but this sentence isn't particularly helpful.

We're only dumping the generated crontab into the gold file here, and the comment makes this sound rather more involved and suggests we're actually running/testing the script ... which presumably already happened long ago.

Reply:
Fixed.